sdk/src/wallet/storage/kind.rs

```diff
@@ -19,11 +22,16 @@ pub enum StorageKind {
 
 impl Default for StorageKind {
     fn default() -> Self {
-        #[cfg(feature = "rocksdb")]
-        return Self::Rocksdb;
-        #[cfg(target_family = "wasm")]
-        return Self::Wasm;
-        #[cfg(not(any(feature = "rocksdb", target_family = "wasm")))]
-        Self::Memory
+        cfg_if::cfg_if!(
+            if #[cfg(feature = "rocksdb")] {
+                Self::Rocksdb
+            } else if #[cfg(feature = "jammdb")] {
+                Self::Jammdb
+            } else if #[cfg(target_family = "wasm")] {
+                Self::Wasm
+            } else {
+                Self::Memory
+            }
+        );
```

Comment:
```suggestion
        )
```

Reply:
The issue still persists. I dont know if we can get away by avoiding those return statements